.github/ISSUE_TEMPLATE/support-request.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Support Request
+about: Request Support
+title: "[SUPPORT REQEUST] CLOSE"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5471038)